### DISCUSSION TITLE
SafeAreaView on stacks

### DISCUSSION THREAD

**Discussion by brunowilliang** [tamagui/tamagui#1701] [Ideas]
It would be nice if we had a property to pass in the stack to get the safe area without needing to import...

instead of this
```js
export default function Page() {
  const { top, bottom } = useSafeAreaInsets()

  return (
    <Stack pt={top} pb={bottom}>
      {/* content */}
    </Stack>
  )
}
```

to this
```js
export default function index() {
  return (
    <Stack safeAreaTop safeAreaBottom>
      {/* content */}
    </Stack>
  )
}
```

Basically, create some utility classes:
SafeArea
safeAreaTop
SafeAreaBottom

**Comment by krishnadubagunta** [comment]
Are there cases where that is not needed? If yes, then does it make sense to include it by default and opt-out when not needed?
